cypress/integration/finance/funds/funds.addBudget.spec.js

```diff
@@ -1,12 +1,14 @@
 import uuid from 'uuid';
 import getRandomPostfix from '../../../support/utils/stringTools';
-import NewFund from '../../../support/fragments/finance/funds/newFund';
+import newFund from '../../../support/fragments/finance/funds/newFund';
 import Funds from '../../../support/fragments/finance/funds/funds';
 import { getCurrentFiscalYearCode } from '../../../support/utils/dateTools';
 import { testType } from '../../../support/utils/tagTools';
 import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import TopMenu from '../../../support/fragments/topMenu';
 
 describe('ui-finance: Add budget to fund', () => {
+  const fundDto = newFund.defaultFund;
```

Comment:
what is Dto into const name? Is it correct?

Reply:
Like a 'data transfer object' : )